lib/buildpackutil.py

```diff
@@ -471,3 +471,6 @@ def bypass_loggregator_logging():
 
 def get_metrics_url():
     return os.getenv("TRENDS_STORAGE_URL")
+
+def is_paid_app():
```

Comment:
I would do this the other way around for a couple of reasons:
- It is possible in principle to modify environment variables, so asking if an app is paid or not is a bit risky (customers could modify this) I think that it is safer to ask "is free app"
- The buildpack is used for applications outside of the Mendix cloud, so we want to design things in a way where restrictions (like this) are opt-in, as opposed to opt-out.

```suggestion
def is_free_app():
    return os.getenv("PROFILE") == "free"
```

Reply:
I thought of this and wasn't sure what method name would be better in the public-facing buildpack codebase. Will change.

Reply:
It's less about naming, and more about gating functionality.

We don't want to _take away_ functionality, unless people opt in. Similarly, we don't want to _enable_ unexpected new functionality, unless people opt in.

We want to preserve things as they are for the most part, and use defaults that reflect this.